chronix-server-query-handler/src/main/java/de/qaware/chronix/solr/query/analysis/functions/FunctionType.java

```diff
@@ -61,18 +61,18 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(FunctionType.class);
 
-    //Sets to hold the aggregations,analyses and transformations.
+    //Sets to hold the aggregations, analyses and transformations.
     //Otherwise the complexity of if(type == X || type == X ...) is to high
-    private static final Set<FunctionType> aggregations = new HashSet<>();
-    private static final Set<FunctionType> analyses = new HashSet<>();
-    private static final Set<FunctionType> transformations = new HashSet<>();
+    private static final Set<FunctionType> AGGREGATIONS = new HashSet<>();
```

Comment:
If they ever invent a static code analysis tools which does not overwhelm you with false positives, please tell me ;)